### PR TITLE
Fix bug when no name was configured

### PR DESCRIPTION
**Pull request by maryum375**
The mail contained "Hello undefined"

### CODE REVIEW

**Comment by eladnava** [comment]
Thanks so much for your contribution @maryum375! 😄 

LGTM. Next time, please format your code according to the project's code style. 👍

**Comment by eladnava** [comment]
Released v1.0.26 @maryum375 👍

File: index.js [hunk 1]
```diff
@@ -113,7 +113,11 @@ Mailgen.prototype.parseParams = function (params) {
 
     // Use `greeting` and `name` for title if not set
     if (!body.title) {
-        body.title = body.greeting + ' ' + body.name + ',';
+        body.title = body.greeting;
```
Comment (lirantal) [inline index.js]: Can't the same thing happen with `body.greeting` being undefined?

Comment (eladnava) [inline index.js]: @lirantal nope, there is already a check for that here:
https://github.com/eladnava/mailgen/blob/461ab1c84ad99488161deb2d71457f2b547e4f56/index.js#L111

Comment (lirantal) [inline index.js]: Ahh cool, didn't open the entire file to look it up.
Good job.

Comment (eladnava) [inline index.js]: @lirantal thanks!

**Pull request merged**